### PR TITLE
Support Bigtable

### PR DESCRIPTION
**Pull request by matthewmturner**

### CODE REVIEW

**Comment by matthewmturner** [comment]
@jychen7 would you be able to test this out with Bigtable (i dont have a google cloud account)?  It should be pretty close.  

FYI this is what my json config file(`~/.datafusion/table_providers/bigtable.json`) looks like:

```
[
	{
		"table_name": "abc",
		"project": "test_project",
		"instance": "test_instance",
		"table": "test_table",
		"column_family": "test_col_family",
		"table_partition_cols": ["col_a", "col_b"],
		"table_partition_separator": "#",
		"utf8_qualifiers": ["abc"],
		"i64_qualifiers": ["xyz"],
		"only_read_latest": true
	}
]
```

**Comment by jychen7** [comment]
it works!!!

I try this branch and `BIGTABLE_EMULATOR_HOST="localhost:8086" cargo run --features bigtable` and using emulator data locally

image

---

yes, Bigtable instance can be expensive ($500/month for one node) and does not support `pause` (as of 2022-03). So my daily development relies on the official bigtable emulator. If checkout `datafusion-contrib/datafusion-bigtable` repo, it can be easily setup via `make bigtable && make seed`
https://github.com/datafusion-contrib/datafusion-bigtable/blob/27df71b290d7b17c5b7b564fbb58b5fb1fe23c8b/Makefile#L1-L9

To connect to real Bigtable instance, use `GOOGLE_APPLICATION_CREDENTIALS=path/to/key.json` instead

File: src/app/datafusion/table_providers/bigtable.rs
Comment (jychen7) [inline src/app/datafusion/table_providers/bigtable.rs:66]: looks like `only_read_latest` is not parsed from JSON

fyi, `only_read_latest` is useful for this schema design, I should add to `datafusion-bigtable` README :D 
https://cloud.google.com/bigtable/docs/schema-design-time-series#new-cells

Comment (matthewmturner) [reply src/app/datafusion/table_providers/bigtable.rs]: thank you for catching that, ive fixed

**Comment by matthewmturner** [comment]
@jychen7 thank you for the help testing.  ive updated, i think this is good to merge for now unless you see anything else needed.  i wont have time to test with the emulator for now - but i do hope to get to it.

once you approve this ill merge to master.  appreciate your help!

**Review by jychen7 — APPROVED** [review]

**Pull request merged**
